### PR TITLE
Translate CVE-2014-8090: Another Denial of Service XML Expansion (zh_cn)

### PR DESCRIPTION
**Pull request by hlcfan**
Related #1074

### CODE REVIEW

**Comment by hlcfan** [comment]
@pmq20 @hlcfan @liuzxc @tzwm @happybai @diguage @alex-marmot @itsliupeng
@cassiuschen @1ternal @killernova @VincentZhao @reyesyang @jun1st @springwq
@Huibean @junzh0u @zigzagpig @42thcoder @GaaraLi @shouxian @johnnyhappy365
@aidewoode @Yunfei1982 @Airmacho @cursorzz @isjia

Please review.

**Comment by JuanitoFatas** [comment]
File name is wrong.

`zh_cn/news/_posts/2014:11:13:rexml-dos-cve-2014-8090.md`

should be

`zh_cn/news/_posts/2014-11-13-rexml-dos-cve-2014-8090.md`

File: zh_cn/news/_posts/2014:11:13:rexml-dos-cve-2014-8090.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+layout: news_post
+title: "CVE-2014-8090：另一个 XML 洪水攻击"
+author: "usa"
+translator: "hlcfan"
+date: 2014-11-13 12:00:00 +0000
+tags: security
+lang: zh_cn
+---
+
+不受限制的实体扩张可导致 REXML 的风险发生，像是这两篇文章所描述的漏洞：[“Entity expansion DoS vulnerability in REXML (XML bomb, CVE-2013-1821)”](https://www.ruby-lang.org/en/news/2013/02/22/rexml-dos-2013-02-22/) 以及 [“CVE-2014-8080：XML 扩张的洪水攻击”](https://www.ruby-lang.org/zh_tw/news/2014/10/27/rexml-dos-cve-2014-8080/)。
```
Comment (JuanitoFatas) [inline zh_cn/news/_posts/2014:11:13:rexml-dos-cve-2014-8090.md]: When `zh_cn` is not availalbe, please link to `en` one, not `zh_tw`, thanks!

File: zh_cn/news/_posts/2014:11:13:rexml-dos-cve-2014-8090.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+这个风险的 CVE 识別号已经被指派为 [CVE-2014-8090](http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2014-8090)。强烈建议您尽快升级 Ruby。
+
+## 细节
+
+这是 [CVE-2013-1821](https://www.ruby-lang.org/en/news/2013/02/22/rexml-dos-2013-02-22/) 和 [CVE-2014-8080](https://www.ruby-lang.org/zh_tw/news/2014/10/27/rexml-dos-cve-2014-8080/) 的额外修复。先前的补丁修正了几处的递归扩展并限制了字符串建立的总长度。但没有针对实体所进行限制。空字符串的递归扩张可能把 CPU 整个吃满。在从 XML 文件读取文字节点时，REXML 解析器可以转成占用极大空间的字符串对象，将机器上的内存用尽，进而导致洪水攻击。
```
Comment (JuanitoFatas) [inline zh_cn/news/_posts/2014:11:13:rexml-dos-cve-2014-8090.md]: same as above comment, link to `en`.

**Comment by hlcfan** [comment]
@JuanitoFatas Updated, please review.

File: zh_cn/news/_posts/2014-11-13-rexml-dos-cve-2014-8090.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    self
+  end
+end
+{% endhighlight %}
+
+## 致谢
+
+感谢 Tomas Hoger 报告这个问题。
+
+## History
```
Comment (springwq) [inline zh_cn/news/_posts/2014-11-13-rexml-dos-cve-2014-8090.md]: `History` need to be translated as well

**Comment by hlcfan** [comment]
@springwq Thank you, this should be good.

File: zh_cn/news/_posts/2014-11-13-rexml-dos-cve-2014-8090.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+layout: news_post
+title: "CVE-2014-8090：另一个 XML 洪水攻击"
+author: "usa"
+translator: "hlcfan"
+date: 2014-11-13 12:00:00 +0000
+tags: security
+lang: zh_cn
+---
+
+不受限制的实体扩张可导致 REXML 的风险发生，像是这两篇文章所描述的漏洞：[“Entity expansion DoS vulnerability in REXML (XML bomb, CVE-2013-1821)”](https://www.ruby-lang.org/en/news/2013/02/22/rexml-dos-2013-02-22/) 以及 [“CVE-2014-8080：XML 扩张的洪水攻击”](https://www.ruby-lang.org/en/news/2014/10/27/rexml-dos-cve-2014-8080/)。
```
Comment (JuanitoFatas) [inline zh_cn/news/_posts/2014-11-13-rexml-dos-cve-2014-8090.md]: ```以及 [“CVE-2014-8080：XML 扩张的洪水攻击”](https://www.ruby-lang.org/en/news/2014/10/27/rexml-dos-cve-2014-8080/)```

If no translation is available, please keep the english title.

**Comment by hlcfan** [comment]
@JuanitoFatas Thank you, please review.

**Comment by JuanitoFatas** [comment]
Merged manually as 10ddcf2. It would be good if you could squash your commits next time! Thanks :boar:

**Pull request closed without merging**